File: src/main/java/block/BlockWater.java
package block;

import registry.RegistryName;

import static org.lwjgl.opengl.GL31.*;

public class BlockWater extends Block {
    public BlockWater() {
        super(new RegistryName("blockgame", "water"));
    }

    @Override
    public boolean isOpaque() {
        return false;
    }

    @Override
    public EnumRenderLayer renderLayer() {
        return EnumRenderLayer.TRANSLUCENT;
    }

    @Override
    public void preRender() {
        glDisable(GL_CULL_FACE);
    }

    @Override
    public void postRender() {
        glEnable(GL_CULL_FACE);
    }
}

File: src/main/kotlin/block/BlockWater.kt
package block

import org.lwjgl.opengl.GL31.*
import registry.RegistryName

class BlockWater : Block(RegistryName("blockgame", "water")) {
    override fun isOpaque(): Boolean {
        return false
    }

    override fun renderLayer(): RenderType {
        return RenderType.TRANSLUCENT
    }

    override fun preRender() {
        glDisable(GL_CULL_FACE)
    }

    override fun postRender() {
        glEnable(GL_CULL_FACE)
    }
}